cmd/airflow.go

```diff
@@ -753,3 +763,13 @@ func prepareDefaultAirflowImageTag(airflowVersion string, httpClient *airflowver
 	}
 	return defaultImageTag
 }
+
+func checkPort(port string) error {
+	host := "localhost:" + port
```

Comment:
Why are we hard coding localhost here? `checkPort()` is being used to check webserver port and postgres port. There is a `config.CFG.PostgresHost` which could have a value that is different than `localhost`.

Reply:
I tried using config.CFG.PostgresHost instead of localhost and that did not work. Only applying "localhost" directly works. config.CFG.PostgresHost is set to postgres by default not sure what that is

Reply:
It is set to "postgres" as default because that value is used by webserver & scheduler container to connect to postgres container, in which case localhost or 0.0.0.0 or 127.0.0.0 does not work for config.CFG.PostgresHost.